File: library/src/main/java/name/zeno/android/util/ObjectUtils.java
package name.zeno.android.util;

/**
 * Create Date: 16/7/18
 *
 * @author 陈治谋 (513500085@qq.com)
 */
public class ObjectUtils
{
  public static <T> boolean equals(T a, T b)
  {
    return (a == null && b == null) || (a != null && b != null && a.equals(b));
  }
}

File: zako/src/main/java/cn/izeno/android/util/ObjectUtils.kt
package cn.izeno.android.util

/**
 * Create Date: 16/7/18
 *
 * @author 陈治谋 (513500085@qq.com)
 */
object ObjectUtils {
  fun <T> equals(a: T?, b: T?): Boolean {
    return a == null && b == null || a != null && b != null && a == b
  }
}
